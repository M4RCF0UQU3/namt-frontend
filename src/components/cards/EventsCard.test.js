import React from 'react';
import Card, { CardActions, CardContent, CardMedia } from 'material-ui/Card';
import MainPageCard from './MainPageCard.jsx';
import Typography from 'material-ui/Typography';
import renderer from 'react-test-renderer';

it('EventsCard correctly', () => {
  const tree = renderer
    .create(<MainPageCard>
				<CardMedia
				  image='/images/voie_ferree.jpg'
				  title="Présentation"
				/>
				<CardContent>
				  <Typography type="headline" component="h3">
				    Faire voyager !
				  </Typography>
				  <Typography component="p">
				    Evénement.
				  </Typography>
				</CardContent>
			</MainPageCard>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});