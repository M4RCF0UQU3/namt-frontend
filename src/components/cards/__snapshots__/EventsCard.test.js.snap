// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventsCard correctly 1`] = `
<div
  className="MuiPaper-root-6 MuiPaper-shadow2-10 MuiPaper-rounded-7"
  style={
    Object {
      "margin": "auto 10px",
      "marginBottom": "20px",
      "minHeight": "400px",
    }
  }
>
  <div
    className="MuiCardMedia-root-33"
    src={undefined}
    style={
      Object {
        "backgroundImage": "url(/images/voie_ferree.jpg)",
      }
    }
    title="Présentation"
  />
  <div
    className="MuiCardContent-root-35"
  >
    <h3
      className="MuiTypography-root-36 MuiTypography-headline-41"
    >
      Faire voyager !
    </h3>
    <p
      className="MuiTypography-root-36 MuiTypography-body1-45"
    >
      Evénement.
    </p>
  </div>
</div>
`;
